### PR TITLE
Pink registration

### PR DESCRIPTION
**Pull request by GuiGou12358**
**Pull Request Summary**

Add `Pink `Asset. I will test the XCM transfers when the token will be available 

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Adds**

Add `Pink `Asset

**Fixes**

- (ex: Fix validation function)

**Changes**

- (ex: Change document B)

**To-dos**

- [x]  test XCM transfers when the token is available

### CODE REVIEW

**Comment by gluneau** [comment]
Hello @GuiGou12358, please provide a publicly available preview URL, see https://docs.astar.network/docs/build/builder-guides/integration_toolings/deploy-astar-portal/#overview

**Comment by GuiGou12358** [comment]
Please find an available preview here: https://astar-portal-pink-asset-5qan80rdw-guigous-projects.vercel.app

**Review by Kahonnohak — APPROVED** [review]
LGTM

**Comment by Kahonnohak** [comment]
@GuiGou12358 
Question - Is it possible to use the gas only DOT? Or prioritising DOT? This was my result (even I had both tokens in both chains)
- DOT token is used for gas fee when you bring PINK token to Astar
- USDT is used for gas fee when you send PINK token to AssetHub

**Comment by GuiGou12358** [comment]
@Kahonnohak 
Normaly, PINK token is used for gas fee when we bring PINK token to Astar
Curently, USDT is used for gas fee when we send PINK token to AssetHub. Here, from my understanding, we can use DOT token or any sufficient asset (USDT or USDC for example).

**Review by impelcrypto — CHANGES_REQUESTED** [review]

File: src/v2/repositories/implementations/XcmRepository.ts [hunk 1]
```diff
@@ -346,4 +357,21 @@ export class XcmRepository implements IXcmRepository {
...
+    destination: XcmChain
+  ): {
+    feeAssetIsRequired: boolean;
+    feeAssetId: string;
+    feeAmount: number;
+  } {
+    if (token.metadata.name === 'PINK' && destination.name === Chain.ASSET_HUB) {
+      // Pink in not a sufficient token and cannot be used as gas fee in Asset Hub,
+      // it means we need to transfer another token to pay the fee and we will use USDT token.
+      return { feeAssetIsRequired: true, feeAssetId: '4294969280', feeAmount: 700000 };
```
[the remark this answers is not in the document]
Comment (GuiGou12358) [reply src/v2/repositories/implementations/XcmRepository.ts]: I did the changes

**Comment by GuiGou12358** [comment]
> @GuiGou12358 Thank you for your explanation.
> 
> > we can use DOT token or any sufficient asset (USDT or USDC for example).
> 
> Yes, I prefer to use DOT instead of USDT for fee. Could you update in this PR?

Sorry, I already did all that for free. Now I'd rather you take over.

**Review by impelcrypto — APPROVED** [review]

**Pull request merged**